cmd/tarmak/cmd/environment_destroy.go

```diff
@@ -0,0 +1,35 @@
+// Copyright Jetstack Ltd. See LICENSE for details.
+package cmd
+
+import (
+	"errors"
+
+	"github.com/jetstack/tarmak/pkg/tarmak"
+	"github.com/spf13/cobra"
+)
+
+// environmentDestroyCmd handles `tarmak environment destroy`
+var environmentDestroyCmd = &cobra.Command{
+	Use:   "destroy [NAME]",
+	Short: "Destroy a environment",
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return errors.New("you have to give the environment name")
+		}
+		globalFlags.Environment.Destroy.Name = args[0]
```

Comment:
We shouldn't need to use global flags here, we pass through the args. We should also check == 1 instead of >0